README.md

```diff
@@ -194,6 +195,15 @@ useEffect(() => {
 
 The list item also extends the `HTMLAnchorElement & HTMLButtonElement` types
 
+### `CommandPalette.FreeSearchAction`
+
+| name           | type                     | required | default    | description                                |
+| -------------- | ------------------------ | -------- | ---------- | -------------------------------------------|
+| index          | number                   | false    | `0`        | Index for list item                        |
+| placeholder    | string                   | false    | `"Search"` | Button placeholder                         |
```

Comment:
I think the default value here should be `"Search for"`, right?

Reply:
I also think the prop should be called something else than placeholder. Probably `prefix`.

Reply:
What do you think about `label`?

Reply:
Label is fine by me

Reply:
Okay, I will change to it.